projects/frontend/shared-components/gui/projects/shared/src/lib/commons/ngx-components/copy-to-clipboard-button/copy-to-clipboard-button.l10n.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
We do not support l10n in the applications consuming these components. 

I guess you can get rid of the support here, as well?

Reply:
I agree; those several l10n.ts files were a part of quickly fetched ngx-component sources.
In case we need to, again, fetch updates of those and sync, then we need to keep the differences or restrain from diverging too much from origin.

Reply:
Bad idea to do this without CI, imho. It requires some refactoring. https://github.com/vmware/versatile-data-kit/issues/1641